src/NServiceBus.Persistence.CosmosDB/Config/ContainerInformation.cs

```diff
@@ -5,7 +5,7 @@
     /// <summary>
     /// Represents the container name and the partition key path when the container information is provided at runtime through the pipeline.
     /// </summary>
-    public readonly struct ContainerInformation
```

Comment:
Ah, looks like the API approval diff we were looking at was misleading, because it appears that `readonly` was already on these two structs!

Reply:
I'll revert the change on those two structs.